wiki/fr/fr-Getting-Started.md

```diff
@@ -0,0 +1,68 @@
+---
+layout: wiki
+title: "Pour commencer"
```

Comment:
Is it really "Pour commencer"? You use "Commencer à utiliser Jamulus" for the h1

Reply:
@ann0see 
> Is it really "Pour commencer"? You use "Commencer à utiliser Jamulus" for the h1

Yes, it is usual in French to contract or summarize in the table of contents.

@Saimyx @trebmuh 
« Pour débuter » for the table of contents (the menu).
Et « Débuter avec Jamulus » for the tittle should be better, no ?

Reply:
> Yes, it is usual in French to contract or summarize in the table of contents.

Ok.

Reply:
> « Pour débuter » for the table of contents (the menu).
> Et « Débuter avec Jamulus » for the tittle should be better, no ?

I've looked at the UI translation. « Premier pas… » (Should be plural imho)
So, i think we must use the same for the doc.

<img width="236" alt="Capture d’écran 2020-11-19 à 07 49 44" src="https://user-images.githubusercontent.com/9108457/99631798-7ee0d400-2a3c-11eb-93fb-b6e67e699fd4.png">

@trebmuh if not done yet for the next release of Jamulus, the link points to the github, not to the "wiki". And of course, it should point to the French version. (i could open an issue on Jamulus repo)